specification/eventhub/resource-manager/Microsoft.EventHub/preview/2022-01-01-preview/namespaces-preview.json

```diff
@@ -0,0 +1,1339 @@
+          "items": {
+            "type": "string"
+          }
+        },
+        "requiredZoneNames": {
+          "type": "array",
+          "description": "The private link resource Private link DNS zone name.",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "PrivateLinkResourcesListResult": {
+      "description": "Result of the List private link resources operation.",
+      "properties": {
+        "value": {
+          "description": "A collection of private link resources",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          }
+        },
+        "nextLink": {
+          "description": "A link for the next page of private link resources.",
+          "type": "string"
+        }
+      }
+    },
+    "userAssignedIdentityProperties": {
+      "x-ms-client-flatten": true,
+      "type": "object",
+      "properties": {
+        "userAssignedIdentity": {
+          "type": "string",
+          "description": "ARM ID of user Identity selected for encryption"
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfigurationList": {
+      "description": "Result of the List NetworkSecurityPerimeterConfiguration operation.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A collection of NetworkSecurityPerimeterConfigurations",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkSecurityPerimeterConfiguration"
+          }
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfiguration": {
+      "description": "Network Security Perimeter related configurations of a given namespace",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NetworkSecurityPerimeterConfigurationProperties",
+          "description": "Properties of the Network Security Perimeter Configuration",
+          "x-ms-client-flatten": true
+        },
+        "id": {
+          "description": "Fully qualified identifier of the resource",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of the resource",
+          "type": "string"
+        },
+        "type": {
+          "description": "Type of the resource",
+          "type": "string"
+        }
+      }
+    },
+    "NetworkSecurityPerimeterConfigurationProperties": {
+      "description": "Properties of NetworkSecurityPerimeterConfiguration",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning state of NetworkSecurityPerimeter configuration propagation",
+          "enum": [
+            "Unknown",
+            "Creating",
+            "Updating",
+            "Accepted",
+            "InvalidResponse",
+            "Succeeded",
+            "SucceededWithIssues",
+            "Failed",
```

Comment:
Add Canceled to be at parity with the ARM RPC

Reply:
Updated